plugins/ProfessionalServices/templates/pluginAdvertising.twig

```diff
@@ -17,10 +17,10 @@
         {% endif %}
     </div>
     <div class="promo-actions">
-        <a class="btn" target="_blank" rel="noreferrer noopener" href="{{ plugin.shop.url|default('')|e('html_attr') }}">
+        <a class="btn" href="?module=Marketplace&action=overview#?showPlugin={{ plugin.name }}">
             {{ 'ProfessionalServices_CTAStartFreeTrial'|translate }}
```

Comment:
Just for completeness: Wouldn't it make sense to display `Request Trial` here as well for non super users?

Reply:
Could possibly create a follow up card for that if not easily added in this round.

Reply:
Created DEV-18027 to check which options we have to take care of for the feature gates, as the plugin may be met with missing requirements and not be available from the start.